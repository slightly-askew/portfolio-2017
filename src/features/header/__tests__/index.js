import React from "react";
import { Header } from "../index";
import renderer from "react-test-renderer";

test("header renders correctly", () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
