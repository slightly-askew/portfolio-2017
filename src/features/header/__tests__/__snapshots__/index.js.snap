// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<div
  className="jhXwPk"
>
  <h1
    className="exSSoJ"
  >
    Oliver Askew
  </h1>
</div>
`;
